test/app/k8s/deployment.json

```diff
@@ -0,0 +1,7 @@
+[
```

Comment:
[q] what is this file for?

Reply:
I was thinking to patch the secret name/image
```
 valueFrom:
              secretKeyRef:
                name:
```